task_processing/plugins/mesos/execution_framework.py

```diff
@@ -470,13 +466,19 @@ def resourceOffers(self, driver, offers):
                     self.task_metadata = self.task_metadata.set(
                         task.task_id.value,
                         md.set(
-                            retries=md.retries + 1,
                             task_state='TASK_STAGING',
                             task_state_ts=time.time(),
                         )
                     )
                     get_metric(TASK_LAUNCHED_COUNT).count(1)
 
+        for reason, items in declined.items():
+            if items:
+                log.info(
+                    "Offers declined because {}: {}".format(reason, items))
```

Comment:
that's kind of the point, otherwise we have line per offer which gets spammy fast